modules/monitoring-managing-user-permissions-using-web-ui.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="managing-user-permissions-using-web-ui_{context}"]
+= Managing user permissions using web UI
+
+This procedure shows how to manage user permissions for monitoring their own services using the web UI.
+
+.Prerequisites
+
+* You need to have a user created.
+* Log in the web console as a cluster administrator.
+
+.Procedure
+
+This example shows how to assign permissions to user `johnsmith` for setting up metrics collection and creating alerting rules in the `ns1` namespace:
+
+. In the web console, navigate to *User Management* -> *Role Bindings* -> *Create Binding*.
+. In *Binding Type*, select the "Namespace Role Binding" type.
+. In *Name*, enter a name for the binding.
+. In *Namespace*, select the namespace where you want to grant the access. In this example, select `ns1`.
+. In *Role Name*, enter `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.
++
+[IMPORTANT]
+====
+Whichever role you choose, you must bind it against a specific namespace as a cluster administrator.
+====
++
+--
+** `monitoring-rules-view` allows reading PrometheusRule custom resources within the namespace.
+** `monitoring-rules-edit` allows creating, modifying, and deleting PrometheusRule custom resources matching the permitted namespace.
+** `monitoring-edit` gives the same permissions as `monitoring-rules-edit`. Additionally, it allows creating new scraping targets for services or pods. It also allows creating, modifying, and deleting ServiceMonitors and PodMonitors.
```

Comment:
s/pods/Pods/
I know we aren't consistent through the entire doc set, but we have guidance for this so calling it out: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

Reply:
Changed "pods" to "Pods" where appropriate in the whole assembly.